packages/button/Button/DirectionalLink.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react"
+import arrowBackward from "@kaizen/component-library/icons/arrow-backward.icon.svg"
+import arrowForward from "@kaizen/component-library/icons/arrow-forward.icon.svg"
+import GenericButton, { GenericProps } from "./components/GenericButton"
+
+type DirectionalLinkProps = Pick<
+  GenericProps,
+  | "id"
+  | "reversed"
+  | "disabled"
+  | "icon"
+  | "onClick"
+  | "onMouseDown"
+  | "href"
+  | "newTabAndIUnderstandTheAccessibilityImplications"
+  | "disableTabFocusAndIUnderstandTheAccessibilityImplications"
+  | "onFocus"
+  | "onBlur"
+> & {
+  label?: string
```

Comment:
I reckon we should make `label` required and remove the defaults. I think I mentioned to you that we've found that people won't do something if they don't need to and the effect isn't immediately obvious. Even if we're not expecting this component to used much, if at all directly, I'd still apply the same ideology. If we make this required, hopefully it will serve as a good reminder to add a label that makes sense for the use case, and hopefully be localised as well.

I'd also be tempted to name this prop `accessibleLabel` to better describe its purpose. My only hesitation is that it would be inconsistent with `IconButton`.